docs/modules/ollama.md

```diff
@@ -19,11 +19,11 @@ go get github.com/testcontainers/testcontainers-go/modules/ollama
 The module allows you to run the Ollama container or the local Ollama binary.
 
 <!--codeinclude-->
-[Creating a Ollama container](../../modules/ollama/examples_test.go) inside_block:runOllamaContainer
+[Creating an Ollama container](../../modules/ollama/examples_test.go) inside_block:runOllamaContainer
 [Running the local Ollama binary](../../modules/ollama/examples_test.go) inside_block:localOllama
 <!--/codeinclude-->
 
-If the local Ollama binary fails to execute, the module will fallback to the container version of Ollama.
+If the local Ollama binary fails to execute, the module will fall back to the container version of Ollama.
```

Comment:
bug: This change is actually incorrect fallback is the noun if other plans fail.

Reply:
I don't follow - I changed it to "fall back" because it's being used as a verb, not a noun.

Reply:
My fail I miss read